src/players/FilePlayer.js

```diff
@@ -78,6 +78,7 @@ export class FilePlayer extends Component {
     if (playsinline) {
       this.player.setAttribute('playsinline', '')
       this.player.setAttribute('webkit-playsinline', '')
+      this.player.setAttribute('x5-playsinline', 'true')
```

Comment:
Does this need to be `true` or can it be `''` like attributes above?

Reply:
'' is ok

Reply:
Ok, let's change it to be consistent and then I'll merge.

```suggestion
      this.player.setAttribute('x5-playsinline', '')
```